packages/lodestar-validator/test/utils/block.ts

```diff
@@ -1,26 +0,0 @@
-import { BeaconBlock } from "@chainsafe/lodestar-types";
```

Comment:
Removed duplicate function `generateEmptyBlock` that is already used from `packages/lodestar`